document-store/src/main/java/org/hypertrace/core/documentstore/model/subdoc/SubDocumentUpdate.java

```diff
@@ -36,7 +46,7 @@ public static SubDocumentUpdate of(final String subDocumentPath, final Boolean[]
   }
 
   public static SubDocumentUpdate of(final String subDocumentPath, final SubDocumentValue value) {
-    return new SubDocumentUpdate(SubDocument.builder().path(subDocumentPath).build(), value);
+    return new SubDocumentUpdate(SubDocument.builder().path(subDocumentPath).build(), SET, value);
```

Comment:
Why are we making the `SET` default? For backward compatibility?

Reply:
Yup. For backward compatibility.